IM_calculation/source_site_dist/src_site_dist.py

```diff
@@ -203,6 +203,9 @@ def calc_rx_ry_GC2(
                 plane_points, [plane_header], np.asarray([loc])
             )
             dists = h_dist_f(plane_points, loc[0], loc[1])
+            # Mimimum distance of 1m to prevent nans/infs
```

Comment:
update the comment to 0.001m or just remove the distance specifier.

Reply:
All distances used in geo and src site dist are in km. 0.001km == 1m

Reply:
the unit matters, i guess thats what jason's comment is about.

Reply:
Oh of course. It's confusing as is with having one number in one unit and just below it's in a different unit.

Reply:
@jasonmotha I'm not sure you understand the code + comment? The code distance is 0.001km, while the comment says 1m. So same distance in different units.

Reply:
well, yes it the same distance, but if I'm debugging and see '1', should i expect it to be 1m or 1km?
so if the value are in km, then the comment should also be in km, or which ever it is as long as it matches.

Reply:
Yeah - what Jonney said.

Reply:
Oh right, github didn't show me your second comment before I posted mine Jason